.evergreen/prepare-shell.sh

```diff
@@ -6,10 +6,9 @@
 
 # This script prepares a shell to run the remaining scripts in this folder
 # It MUST be kept idempotent! It will overwrite the orchestration config and expansion.yml file upon every run
-# and it will only clone drivers-tools if they do not exist one directory above our driver src
 
 PROJECT_DIRECTORY="$(pwd)"
-DRIVERS_TOOLS=$(cd .. && echo "$(pwd)/drivers-tools")
+DRIVERS_TOOLS="$(pwd)/drivers-evergreen-tools"
```

Comment:
I was imagining that as a part of this work, we'd remove any references to `DRIVERS_TOOLS` in our evergreen tooling because we now always have DET installed at the same path. 

Is this a change you could make as well?

Reply:
I was hesitant to make this change as the diff would be incredibly large. I see really no harm in keeping the environment variable because it means it's one place to change the value instead of hundreds in the future if we need to. If you don't mind I'd prefer to keep it.

Reply:
Yeah, that's fine.